reservations/tests/test_allocation_model.py

```diff
@@ -115,11 +123,28 @@ def test_mapping_application_round_baskets(
     recurring_application_event,
 ):
 
-    data = AllocationData(application_round=application_round_with_reservation_units)
-    assert data.baskets == {
-        application_round_basket_one.id: application_round_basket_one,
-        application_round_basket_two.id: application_round_basket_two,
-    }
+    data = AllocationDataBuilder(
+        application_round=application_round_with_reservation_units
+    ).get_allocation_data()
+
+    assert_that(data.baskets).contains_key(
+        application_round_basket_one.id, application_round_basket_two.id
+    )
+
+    assert_that(data.baskets[application_round_basket_one.id]).has_id(
+        application_round_basket_one.id
+    ).has_order_number(
```

Comment:
this dynamic assertion is nice :sunglasses:

Reply:
Yeah, i kind of find magic like this a bit disturbing, could be hard to debug, but seems to work.